src/accessibility/roving/RovingAccessibleTooltipButton.tsx

```diff
@@ -16,26 +16,23 @@ limitations under the License.
 
 import React, { ComponentProps } from "react";
 
-import AccessibleTooltipButton from "../../components/views/elements/AccessibleTooltipButton";
 import { useRovingTabIndex } from "../RovingTabIndex";
 import { Ref } from "./types";
+import AccessibleButton from "../../components/views/elements/AccessibleButton";
 
-type Props<T extends keyof JSX.IntrinsicElements> = Omit<
-    ComponentProps<typeof AccessibleTooltipButton<T>>,
-    "tabIndex"
-> & {
+type Props<T extends keyof JSX.IntrinsicElements> = Omit<ComponentProps<typeof AccessibleButton<T>>, "tabIndex"> & {
     inputRef?: Ref;
 };
 
-// Wrapper to allow use of useRovingTabIndex for simple AccessibleTooltipButtons outside of React Functional Components.
+// Wrapper to allow use of useRovingTabIndex for simple AccessibleButtons outside of React Functional Components.
 export const RovingAccessibleTooltipButton = <T extends keyof JSX.IntrinsicElements>({
```

Comment:
Shouldn't we also inline this into RovingAccessibleButton then?

Reply:
As describe in the PR description, I will replace the usage of `RovingAccessibleTooltipButton` by `RovingAccessibleButton` in a next PR. Just focusing currently in the `AccessibleTooltipButton` migration.